cli/command/image/remove.go

```diff
@@ -56,9 +57,13 @@ func runRemove(dockerCli command.Cli, opts removeOptions, images []string) error
 	}
 
 	var errs []string
+	var removeConflict = false
 	for _, img := range images {
 		dels, err := client.ImageRemove(ctx, img, options)
 		if err != nil {
+			if apiclient.IsErrImageRemoveConflict(err) {
+				removeConflict = true
+			}
```

Comment:
I think we should invert this. Any `NotFound` can be skipped in the force case, but any other errors should still be reported as errors.